### PR TITLE
Add user research manifest file

### PR DESCRIPTION
**Pull request by danmitchell-**
### Context
We want to have a staging and user research environment

### Changes proposed in this pull request
Add manifest for staging and research environment on paas

### Guidance to review
`cf target -s target sandbox`
`cf push manifest-#{ENVIRONMENT}.yml`

### CODE REVIEW

**Comment by gidsg** [comment]
I wonder if the research environment only needs 1 instance of `registers-frontend` as we could tolerate downtime during a deployment?
Also, do we definitely want a `staging` environment. I'm not opposed for ad-hoc testing but think it worth asking the question.

**Comment by michaelabenyohai** [comment]
I agree that I'm not sure we need a staging environment.

So we have thoughts around which PaaS space we will put the research environment in? I would guess not prod.

**Comment by danmitchell-** [comment]
I was going to put research in `sandbox` @michaelabenyohai, do you agree?

**Comment by michaelabenyohai** [comment]
Yeah I think `staging` space makes sense. 

Just had a thought about whether we need the research environment to have it's own database, schedulers and queues? Maybe it's worth only deploying the frontend app against the same database for now. If there is a later requirement for research to be able to add their own registers or register data we can update the manifest then?

**Comment by danmitchell-** [comment]
@michaelabenyohai personally I would have a separate database for the research environment so they can play with that as much as they like without having any effect to production. I don't think we can share schedulers and queues because the app would be in a different `space`.

**Comment by michaelabenyohai** [comment]
Good point, if we share the database we would have to put this in prod environment. Let's go with creating all the apps then.

**Comment by michaelabenyohai** [comment]
I'm happy with this other than potentially addressing @gidsg comment about the UR environment only requiring 1 instance - what do you think @danmitchell- ?

File: manifest-production.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+
+inherit: manifest-base.yml
+domain: registers-trial.service.gov.uk
```
Comment (michaelabenyohai) [inline manifest-production.yml]: Is this section specific to the `registers-frontend` application? The inherited `manifest-base.yml` contains 3 applications so does this definitely override the correct one?

**Pull request closed without merging**